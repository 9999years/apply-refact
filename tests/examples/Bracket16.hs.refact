[("tests/examples/Bracket16.hs:1:8: Warning: Redundant bracket\nFound:\n  do f\n     (print x)\nWhy not:\n  do f\n     print x\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 1, startCol = 14, endLine = 1, endCol = 23}, subts = [("x",SrcSpan {startLine = 1, startCol = 15, endLine = 1, endCol = 22})], orig = "x"}])]
