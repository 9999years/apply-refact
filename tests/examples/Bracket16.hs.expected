main = do f; print x
